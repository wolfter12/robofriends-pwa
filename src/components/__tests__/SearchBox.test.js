import React from 'react';
import { shallow } from 'enzyme';

import SearchBox from '../SearchBox';

it('expect to render SearchBox component', () => {
  const wrapper = shallow(<SearchBox />);

  expect(wrapper.length).toEqual(1);
  expect(wrapper).toMatchSnapshot();
});
