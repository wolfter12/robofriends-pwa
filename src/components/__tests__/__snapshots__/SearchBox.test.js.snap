// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SearchBox component 1`] = `
<div
  className="pa2"
>
  <input
    aria-label="Search"
    className="pa3 ba b--green bg-lightest-blue"
    placeholder="search robots"
    type="search"
  />
</div>
`;
